components/mdx/ModalImage.tsx

```diff
@@ -2,7 +2,9 @@ import React, { useState } from 'react'
 import Modal from 'react-bootstrap/Modal'
 import Image, { ImageProps } from 'next/image'
 
-const ModalImage: React.FC<ImageProps> = (props: ImageProps) => {
+const ModalImage: React.FC<
+  ImageProps & { long?: string | boolean; tall?: string | boolean }
```

Comment:
Why `string | boolean`? With only boolean you don't need to pass a value to it like you needed.

Reply:
That React warning is not related to typescript, it comes from passing boolean as an attribute to element. Maybe I'm blind but I can't figure out why it's happening.